src/main/java/uk/gov/digital/ho/pttg/application/namematching/candidates/AbbreviatedNamesFunctions.java

```diff
@@ -12,25 +12,26 @@
 public final class AbbreviatedNamesFunctions {
 
     private static final String ANY_LETTER_INCLUDING_UNICODE_MATCHER = "\\p{L}\\p{M}*+";
-    private static final String FULL_STOP_SPACE_MATCHER = "\\.\\s+";
-    private static final String FULL_STOP_SPACE_BETWEEN_NAMES_PATTERN = ANY_LETTER_INCLUDING_UNICODE_MATCHER + FULL_STOP_SPACE_MATCHER + ANY_LETTER_INCLUDING_UNICODE_MATCHER;
-    private static final Pattern FULL_STOP_SPACE_PATTERN_REGEX = Pattern.compile(FULL_STOP_SPACE_BETWEEN_NAMES_PATTERN);
+    private static final String ABBREVIATION_SPACE_MATCHER = "([.'])\\s+";
```

Comment:
This looks a lot better - but actually brackets "(" and ")" are superfluous. The regex
[.']\\s+
works just as well.